subcommand/inject-connect/command.go

```diff
@@ -351,8 +346,14 @@ func (c *Command) Run(args []string) int {
 	allowK8sNamespaces := flags.ToSet(c.flagAllowK8sNamespacesList)
 	denyK8sNamespaces := flags.ToSet(c.flagDenyK8sNamespacesList)
 
-	// Start the endpoints controller
-	zapLogger := zap.New(zap.UseDevMode(true), zap.Level(zapcore.InfoLevel))
+	var zapLevel zapcore.Level
+	if err := zapLevel.UnmarshalText([]byte(c.flagLogLevel)); err != nil {
+		c.UI.Error(fmt.Sprintf("Error parsing -log-level %q: %s", c.flagLogLevel, err.Error()))
+		return 1
+	}
```

Comment:
does zaplogger support the same levels as hclog?

Reply:
good catch..trace is not supported!! updating the PR!

Reply:
Updated the flag description to only include the levels supported by zaplogger!